internal/fuzzbinder/fuzzclaim.go

```diff
@@ -180,3 +181,122 @@ func parseFuzzClaimBytes(statementBytes []byte) (*intoto.Statement, error) {
 
 	return &statement, nil
 }
+
+// generateFuzzClaimSpec generates a fuzzing claim specification using the
+// fuzzing reports of OSS-Fuzz.
+func generateFuzzClaimSpec(date string, revisionDigest intoto.DigestSet, fuzzParameters *FuzzParameters, fuzzTargets []string) (*FuzzClaimSpec, error) {
```

Comment:
This function is too long and does too many things. Can you split into smaller functions?
There is a long distance between variable declaration and where they are used (e.g., in case of projectCoverage). 
I suggest you move the two for-loops for perTarget coverage into their own functions, and call them here for better readability.

Reply:
I added a related issue to split  this function `// TODO(#171): Split generateFuzzClaimSpec into smaller functions.`